azure-pipelines.release-fluentui.yml

```diff
@@ -99,17 +99,16 @@ extends:
                 timeoutInMinutes: 2
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30251&commitId=e4d617648d87a7f5ed3bd0023e451c7efd6756a7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.customSize+image.normal.chromium.png|761| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|557| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>